scram-tools.file/tools/intel-vtune/intel-vtune.xml

```diff
@@ -1,7 +1,7 @@
 <tool name="intel-vtune" version="@TOOL_VERSION@">
   <info url="https://www.intel.com/content/www/us/en/developer/tools/oneapi/vtune-profiler.html"/>
   <client>
-    <environment name="INTEL_VTUNE_BASE" default="@TOOL_ROOT@"/>
+    <environment name="INTEL_VTUNE_BASE" default="@INTEL_VTUNE_INSTALLDIR@"/>
```

Comment:
is this needed because of `NO_AUTO_DEPENDENCY` ?

Reply:
No, not due to `NO_AUTO_DEPENDENCY`.  `TOOL_ROOT` points to the actual install path of a package in this case it will be `/our/cvmfs/install/dir/<arch>/externals/inten-vtune/<version>` but we want `INTEL_VTUNE_BASE` to point to hardcoded `/cvmfs/projects.cern.ch/intelsw/oneAPI/....` path. One way to achieve this is to override `TOOL_ROOT` in the `etc/prodile.d/init.sh` e.g. we could have done the following the intel-vtune.spec `%post`
```
echo "INTEL_VTUNE_ROOT=''/cvmfs/projects.cern.ch/intelsw/oneAPI/..." > $RPM_INSTALL_PREFIX/%{pkgrel}/etc/profile.d/init.sh
```

but as packaging system also use `<TOOL>_ROOT` to I would avoid pointing  `<TOOL>_ROOT` to something else. That is why spec file now saves `INTEL_VTUNE_INSTALLDIR /cvmfs/projects.cern.ch/intelsw/oneAPI/linux/x86_64/%{year}/vtune/%{realversion}` int he init file which is then used by xml file